mitxgraders/__init__.py

```diff
@@ -12,6 +12,13 @@
 # Add the current version
 from mitxgraders.version import __version__
 
+# Add voluptuous
+try:
+    import voluptuous
+except ImportError:
+    raise ImportError("External dependency 'voluptuous' not found;"
```

Comment:
Oh no, the coverage! 

So after a bit of research, we can test those lines with:
```python
import sys
import mock
from pytest import raises
import mitxgraders

def test_voluptuous_import_error_message():
    with mock.patch.dict(sys.modules, {'voluptuous': None}):
        msg = ("External dependency 'voluptuous' not found;"
               " see https://github.com/mitodl/mitx-grading-library#faq")
        with raises(ImportError, match=msg):
            # importing is not good enough to raise the error; need to reload
            # because conftest.py already imported voluptuous
            reload(mitxgraders)
```
(I installed `mock` a while back for testing comparer functions anyway).

This was perhaps more effort than it was worth, but fun to learn :).

Reply:
I needed to import reload from imp, but other than that, seems to work!